subsys/net/lib/sockets/CMakeLists.txt

```diff
@@ -7,6 +7,7 @@ zephyr_sources(
   sockets_select.c
```

Comment:
In commit "net: sockets: Add socket api to support AF_PACKET", there is difficult to understand sentence.
Could we say here:

```
This commit adds packet socket support to socket api. This version supports
basic packet socket features. Protocol family is AF_PACKET, type
of socket is SOCK_RAW and proto type is ETH_P_ALL. The user
will receive every packet (with L2 header) on the wire.
For TX, the subsystem expects that the user has set all the protocol headers (L2 and L3) properly.
```